pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1321,6 +1321,12 @@ type ObjectStoreSpec struct {
 	// +optional
 	// +nullable
 	Security *SecuritySpec `json:"security,omitempty"`
+
+	// Network related configuration
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +nullable
+	// +optional
+	Network NetworkSpec `json:"network,omitempty"`
```

Comment:
Can you also update the default examples in cluster.yaml and the helm charts.

Reply:
no needed to update cluster.yaml